Calmspace/DetailsTableViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  DetailsTableViewController.swift
+//  Calmspace
+//
+//  Created by Admin on 12/10/2023.
+//
+
+import UIKit
+
+struct PracticeDetail {
+    let title: String
+    let youtubeLink: URL
+}
+
+class DetailsTableViewController: UITableViewController {
+    
+    var selectedArray: [PracticeDetail] = []
```

Comment:
`selectedArray` — does it contain selection? Or all the objects?
If it is selection only, the variable can be called `selection` (but I do not see any selection in this table). 
If it contains all the objects, you can call it `meditations` or something like it.